### DISCUSSION TITLE
Could we allow for DRY chaining of classes after arbitrary variants?

### DISCUSSION THREAD

**Discussion by geoyws** [tailwindlabs/tailwindcss#11967] [Ideas]
e.g.: 
`[&>span]:underline [&>span]:cursor-pointer`

could instead be just

`[&>span]:underline:cursor-pointer`
or 
`[&>span]:(underline,cursor-pointer)`

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)

**Reply by geoyws** [reply]
https://play.tailwindcss.com/D4ex5ghgKB
Going to try [this](https://github.com/tailwindlabs/tailwindcss/discussions/11701#discussioncomment-6569866) and [this](https://github.com/tailwindlabs/tailwindcss/discussions/11701#discussioncomment-6633770) first and see what happens. Apologies for the numerous edits!

Although I must add that logically... one would expect to DRY and the default situation we have is quite odd :(

**Reply by geoyws** [reply]
So perhaps could we consider, in the spirit of utility, that the class `[&>span]:(underline,cursor-pointer)` in the TSX will be ultimately `[&>span]:underline [&>span]:cursor-pointer` when in the DOM? This way there will be no issues with overriding with utilities, duplication and etc. Might there be any other concerns that I might be ignorant of?
